res/values-b+es+r419/strings.xml

```diff
@@ -74,50 +71,49 @@
     <string name="OK">"VALE"</string>
     <string name="CANCEL">"CANCELAR"</string>
     <string name="Chat_Name">"Nombre del chat"</string>
-    <string name="Max_People">"Personas máximas"</string>
+    <string name="Max_People">"Jugadores máximos"</string>
     <string name="START">"EMPEZAR"</string>
     <string name="CREATE">"CREAR"</string>
     <string name="CUSTOM_GAMES">"PARTIDAS PERSONALIZADAS"</string>
     <string name="CHAT">"CHAT"</string>
     <string name="CUSTOM_SETUP">"CONFIGURACIÓN PERSONALIZADA"</string>
     <string name="CHAT_SETUP">"CONFIGURACIÓN DEL CHAT"</string>
 
-
     <string name="SIGN_IN">"INICIAR SESIÓN"</string>
     <string name="ACCOUNT">"CUENTA"</string>
     <string name="Red_Team">"Equipo Rojo"</string>
     <string name="Green_Team">"Equipo Verde"</string>
     <string name="Blue_Team">"Equipo Azul"</string>
     <string name="Yellow_Team">"Equipo Amarillo"</string>
-    <string name="Signed_in_as_">"Conectado como:"</string>
-    <string name="Not_signed_in_">"¡No registrado!"</string>
+    <string name="Signed_in_as_">"Conectado como"</string>
+    <string name="Not_signed_in_">"¡No has iniciado sesión!"</string>
     <string name="SET_NAME">"CAMBIAR NOMBRE"</string>
     <string name="SIGN_OUT">"CERRAR SESIÓN"</string>
     <string name="ACHIEVEMENTS">"LOGROS"</string>
     <string name="STATISTICS">"ESTADÍSTICAS"</string>
-    <string name="XP_">"EXPERIENCIA:"</string>
+    <string name="XP_">"EXP"</string>
 
     <string name="Information">"Información"</string>
-    <string name="To_use_this_skin_you_must_earn_the_achievement">"Para utilizar esta skin tienes que completar el logro"</string>
-    <string name="To_use_this_skin_you_must_reach_level">"Para utilizar esta skin tienes que alcanzar el nivel"</string>
+    <string name="To_use_this_skin_you_must_earn_the_achievement">"Para utilizar esta apariencia, debes completar el logro"</string>
+    <string name="To_use_this_skin_you_must_reach_level">"Para utilizar esta apariencia, debes alcanzar el nivel"</string>
 
-    <string name="Single_Player_Stats">"Estadísticas en modo "Un Jugador""</string>
+    <string name="Single_Player_Stats">"Estadísticas en modo un jugador"</string>
 
     <string name="Level">"Nivel"</string>
-    <string name="PAUSED">"EN PAUSA"</string>
-
+    <string name="PAUSED">"Pausado"</string>
 
     <string name="FRIENDS">"AMIGOS"</string>
     <string name="Warning">"Advertencia"</string>
     <string name="SIGNING_IN">"INICIANDO SESIÓN…"</string>
-    <string name="New_Friend_Request">"¡Nueva solicitud de amistad!"</string>
+    <string name="New_Friend_Request">"¡Tienes una nueva solicitud de amistad!"</string>
     <string name="REQUEST_FRIEND">"SOLICITUD DE AMISTAD"</string>
     <string name="Private">"Privado"</string>
     <string name="Public">"Público"</string>
     <string name="Bot">"Bot"</string>
-    <string name="You">"Usted"</string>
+    <string name="You">"Tú"</string>
```

Comment:
For Latin America, according to my knowledge, "Usted" is the formal way, so the change to this one is not really needed.